portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,10 @@ function addRandomFact() {
   const factContainer = document.getElementById('fact-container');
   factContainer.innerText = fact;
 }
+
+// Add greeting to the page
```

Comment:
I think function spec has to be block /** */ comments instead of the forward slashes